docs/README.md

```diff
@@ -9,70 +7,49 @@
 
 -->
 
-## Installation
-
-### Using pre-built releases
-
-#### Using the `packer init` command
+### Installation
 
-Starting from version 1.7, Packer supports a new `packer init` command allowing
-automatic installation of Packer plugins. Read the
-[Packer documentation](https://www.packer.io/docs/commands/init) for more information.
-
-To install this plugin, copy and paste this code into your Packer configuration .
-Then, run [`packer init`](https://www.packer.io/docs/commands/init).
+To install this plugin, copy and paste this code into your Packer configuration, then run [`packer init`](https://www.packer.io/docs/commands/init).
 
 ```hcl
 packer {
   required_plugins {
-    name = {
-      version = ">= 0.0.1"
-      source  = "github.com/hashicorp/name"
+    amazon = {
+      # source represents the GitHub URI to the plugin repository without the `packer-plugin-` prefix.
+      source  = "github.com/organization/plugin-name"
+      version = "~> 1"
```

Comment:
Should we replace it with a `~>` version specifier? I'd think that people that will fork this will not immediately release v1+ of their plugins.
The old specifier should work, no?